engine/include/gen/graphics/swapchain.hpp

```diff
@@ -39,12 +39,12 @@ namespace gen
 		GEN_NODISCARD vk::SwapchainKHR const & getSwapChain() const { return m_swapChain.get(); }
 		GEN_NODISCARD vk::Extent2D const & getExtent() const { return m_swapChainInfo.imageExtent; }
 		GEN_NODISCARD vk::Format const & getFormat() const { return m_swapChainInfo.imageFormat; }
-		GEN_NODISCARD std::span<vk::Image> const & getImages() const { return m_swapChainImages; }
-		GEN_NODISCARD std::span<vk::UniqueImageView> const & getImageViews() const { return m_swapChainImageViews; }
+		GEN_NODISCARD std::span<vk::Image const> const & getImages() const { return m_swapChainImages; }
+		GEN_NODISCARD std::vector<vk::UniqueImageView> const & getImageViews() const { return m_swapChainImageViews; }
```

Comment:
Shouldn't this return a span too?

Reply:
Removed the use of span here as it was giving me seg faults with vulkan. Will have to go back to it later with a fine tooth comb to find out why this was the case. (Should add I've effectively removed all use of span in the image code due to having multiple seg faults)